modules/tinymce/changelog.txt

```diff
@@ -5,6 +5,7 @@ Version 5.6.0 (TBD)
     Added new `closest` formatter API to get the closest matching selection format from a set of formats. #TINY-6479
     Added new `name` field to the `style_formats` setting object to enable specifying a name for the format. #TINY-4239
     Changed `readonly` mode to allow hyperlinks to be clickable #TINY-6248
+    Fixed an exception thrown when using Shadow DOM caused by attempting to call `.parentNode` on an object which has no parent #TINY-6540
```

Comment:
I am wondering if it should mention `DomUtils.getParents` somewhere?

Reply:
Yeah, this needs some rewording as the exception wasn't from calling `.parentNode` is wasn't even in `DOMUtils`. Maybe something like this?
```suggestion
    Fixed `DOMUtils.getParents` incorrectly including the shadow root in the array of elements returned #TINY-6540
```